### ISSUE TITLE
Clashes detected. Ignore-name-clashes/fix-clashes doesn't seem to solve problem

### ISSUE THREAD

**Issue by marcederis** [odeke-em/drive#457]
Hello,

I'm trying to pull only one specific folder from my Google Drive called 'Recent'.
When I run `drive pull Recent -ignore-name-clashes` or `drive pull Recent -fix-clashes`, it says, "Resolving..." and then repeats the message, "These paths clash" followed by a list of files that clash. Finally, it repeats the message "clashes detected. Use `ignore-name-clashes`..." again, even though I'm already using either the "ignore" or "fix" flag and neither appears to have done anything.
I'm using drive for the first time, so I haven't managed to pull any files down yet. Any ideas, or is there something I needed to do besides the `drive init` step?
One thing I should note is that I'm trying to use drive in a FreeBSD jail, so I don't know whether that has anything to do with my issues pulling files? I don't get any errors otherwise, except for the occasional googleapi error 500 or 503, but I think those are intermittent api errors not specifically to do with drive.
This is what I get from "version":
```
$ drive version
drive version: 0.3.2
Commit Hash: 'dc7934a3df8ee0b4daf97e66aeed5bb892adc9aa'
Go Version: go1.5.1
OS: freebsd/amd64
BuildTime: 2015-11-04 13:30:44.168160006 -0800 PST
```

Thanks for any help!
-Marc

**Comment by odeke-em** [comment]
Hello there @marcederis!
Thank you for reporting this issue and welcome to drive.

So off the top, I noticed that you said
```shell
$ drive pull Recent --fix-clashes
```
Please make that
```shell
$ drive pull --fix-clashes Recent
```
Unfortunately this is a limitation with Go's flag parser but in the future I see hope in fixing it.
That should help alleviate the problem.

This is awesome, you the first person I've met who properly compiled drive with drive-gen and provided drive version without me having to ask :) -- thanks.

Please let me know if passing in the args before the options helps and then we can proceed debugging this issue.

**Comment by marcederis** [comment]
Hi Odeke-em,

Ah okay thanks! That actually solved the problem. I was able to proceed to the actual downloading step.
As for the "drive-gen" part, haha yes I was going through previous issues to see if my question had already been asked when I noticed you had requested that command in a number of them. I thought I would preempt you having to ask for it :)

Anyway, seems my problem is solved. Thanks for making this project available by the way. It was really hard finding a Google Drive project that would compile/work on FreeBSD so this is great!
